odbc/src/api/definitions.rs

```diff
@@ -98,11 +110,13 @@ pub enum RetrieveData {
 pub enum RowOperationPtr {}
 
 #[derive(Clone, Copy, Debug)]
+#[repr(i32)]
 pub enum SimulateCursor {
     NonUnique = 0,
 }
 
 #[derive(Clone, Copy, Debug, FromPrimitive)]
+#[repr(i32)]
```

Comment:
The specs says SQLULEN, but the values in the ODBC headers (Windows, unixodbc, iodbc) are unsigned int.
```suggestion
#[repr(u32)]
```